symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -163,12 +164,17 @@ protected static function saveFieldData($entry_id, $field_id, $field)
                 Symphony::Database()->insert($fields, $table_name);
             }
         } catch (Exception $ex) {
+            $exception = $ex;
             Symphony::Log()->pushExceptionToLog($ex, true);
         }
 
         if ($did_lock) {
             Symphony::Database()->query('UNLOCK TABLES');
         }
+
+        if ($exception) {
+            throw $exception;
```

Comment:
This would then move into the catch.

Reply:
`finally` is PHP 5.5 and later only.

Reply:
Ah, ignore me then. Forgot we're still supporting PHP 5.3 :)

Reply:
Yup @brendo I've made that mistake to often. And `[]` also. Urg!